Source/Core/Core/Movie.cpp

```diff
@@ -803,6 +803,8 @@ void CheckPadStatus(const GCPadStatus* PadStatus, int controllerID)
 
   s_padState.is_connected = PadStatus->isConnected;
 
+  s_padState.get_origin = ((PadStatus->button & PAD_GET_ORIGIN) != 0);
```

Comment:
Outer parentheses are superfluous, since it's an assignment. LGTM otherwise.

Reply:
I just copied the semantic from the other lines. Wasn't sure if it was there for readability or something.